geth-utils/gethutil/mpt/witness/gen_witness_transactions_test.go

```diff
@@ -16,11 +16,54 @@ import (
 )
 
 func TestTransactions(t *testing.T) {
-	t.Skip("failing test")
 	txs := make([]*types.Transaction, 70)
 	key, _ := crypto.GenerateKey()
 	signer := types.LatestSigner(params.TestChainConfig)
 
+	/*
+		This test inserts 70 transactions into a stacktrie. When the first transaction is
+		added, a leaf is added to the trie (it has index 1 which is used as a key when
+		inserting into a trie, the key is changed by calling KeybytesToHex(key) in TryUpdate
+		to [0, 1, 16]).
+
+		When the second transaction is added (it has index 2, when inserting into a trie,
+		it gets changed to [0, 2, 16]), the extension node E is created with nibble 0 (note that
+		both leaves/transactions have the first nibble 0) and the underlying branch B with children
+		at positions 1 and 2 (second nibble of the two leaves).
+
+		At this point, the proof for the second transaction is (proofC when index = 2):
+		[226 16 160 212 52 159 164 192 63 244 122 229 5 208 58 20 16 54 62 169 98 62 238 163 88 174 155 252 118 132 91 148 62 122 23]
+		[248 81 128 160 32 244 75 78 180 11 251 73 229 254 70 16 254 170 54 254 200 97 231 24 180 34 220 244 153 76 1 194 23 63 64 224 160 46 141 2 37 188 204 110 232 46 31 230 82 226 213 98 71 18 241 37 90 213 167 221 58 33 36 249 248 180 207 235 47 128 128 128 128 128 128 128 128 128 128 128 128 128 128]
+		[248 200 2 131 4 147 224 131 1 226 65 148 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 4 184 100 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 37 160 163 65 77 46 20 3 175 162 34 86 182 41 225 43 90 92 158 249 153 67 193 148 178 63 8 81 26 169 176 56 242 78 160 21 37 82 209 254 5 113 171 235 198 244 52 17 233 162 51 76 151 85 224 28 101 146 160 197 216 253 237 240 187 19 184]
+
+		Note that the first proof element is an extension node with nibble 0 = 16 - 16 (see
```

Comment:
This explanation is really nice and clear, and I wish I could have them before I started this issue. 😭